projects/storefrontapp-e2e-cypress/cypress/helpers/update-password.ts

```diff
@@ -65,7 +65,6 @@ export function updatePassword() {
   cy.visit('/login');
   login(standardUser.registrationData.email, newPassword);
   cy.get(helper.userGreetSelector).should('exist');
```

Comment:
Line 13 and 24, similar to what I said in address book and update email

Reply:
for the standard user, because it re-uses the data again later...